Modelica/Mechanics/Translational/Components/IdealGear.mo

```diff
@@ -0,0 +1,69 @@
+within Modelica.Mechanics.Translational.Components;
+model IdealGear "Ideal gear without mass"
+  extends Modelica.Mechanics.Translational.Interfaces.PartialElementaryTwoFlangesAndSupport2;
+  parameter Real ratio(start=1)
+    "Transmission ratio (flange_a.s/flange_b.s)";
+
+equation
+  if abs(ratio) < Modelica.Constants.eps then
```

Comment:
I understand that such a ratio likely leads to problems, but I don't think that these equations are a good idea.
I would either:
- Just not do anything
- Have an assert instead.